src/web/app/pages-instructor/instructor-student-records-page/instructor-student-records-page.component.spec.ts

```diff
@@ -50,4 +50,8 @@ describe('InstructorStudentRecordsPageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default fields', () => {
+    expect(fixture).toMatchSnapshot();
```

Comment:
We can keep the snapshot. In addition,
I think now you can
1. inject a fake profile data
2. assert the url match your expection

Reply:
I tried doing this in the past, but it seems ng-reflect has a character cap, so no we won't be able to see the full expression. https://github.com/angular/angular/issues/28786

Reply:
I'm not quite sure how to inject the queryParams info into the subscribe callback without mocking it, but it works with undefined queryParams